vbd/oncho_2/app.R

```diff
@@ -0,0 +1 @@
+odin.ui:::odin_basic("model.R", show_table_summary = TRUE)
```

Comment:
should be `oncho2` not `oncho_2`, though this is not terribly clear. Perhaps there is a better naming system here, happy to discuss